public/lib/commands/sessions/push.liquid

```diff
@@ -0,0 +1,5 @@
+{% liquid
```

Comment:
I think `commands/sessions/set.liquid` would be a better name, so it'd be consistent.

Reply:
but otherwise ✅

Reply:
also, this feels more like a `set` than a `push`

Reply:
Fixed.